packages/mds-transaction-api/@types/index.ts

```diff
@@ -23,7 +23,11 @@ export const [TRANSACTION_API_DEFAULT_VERSION] = TRANSACTION_API_SUPPORTED_VERSI
 // Allow adding type definitions for Express Request objects
 export type TransactionApiRequest<B = {}> = ApiRequest<B>
 
-export type TransactionApiAccessTokenScopes = 'transactions:read' | 'transactions:read:claim' | 'transactions:write'
+export type TransactionApiAccessTokenScopes =
+  | 'transactions:read'
+  | 'transactions:read:provider'
+  | 'transactions:read:claim'
```

Comment:
I think `transactions:read:claim` isn't being used and was probably meant to indicate that transactions should be filtered by claims, such as `provider_id` and perhaps one day `agency` or `jurisdiction`, however, I think you should proceed with adding `transactions:read:provider`, but remove `transactions:read:claim`.